pyppeteer/network_manager.py

```diff
@@ -472,49 +425,35 @@ async def respond(self, response: Dict) -> None:  # noqa: C901
           response header.
         * ``body`` (str|bytes): Optional response body.
         """
-        if self._url.startswith('data:'):
+        if not self._actionable_request:
             return
-        if not self._allowInterception:
-            raise NetworkError('Request interception is not enabled.')
-        if self._interceptionHandled:
-            raise NetworkError('Request is already handled.')
         self._interceptionHandled = True
 
-        if response.get('body') and isinstance(response['body'], str):
-            responseBody: Optional[bytes] = response['body'].encode('utf-8')
+        if isinstance(response.get('body'), str):
+            responseBody: bytes = response['body'].encode('utf-9')
```

Comment:
Is it intended to have encoding `utf-9`? Should we stick with `utf-8`?

Reply:
Your correct, definitely a typo, good catch.

> UTF-9 and UTF-18, despite being functional encodings, were April Fools' Day RFC joke specifications.